Content.Shared/Contraband/ContrabandSystem.cs

```diff
@@ -68,17 +81,23 @@ private void OnExamined(Entity<ContrabandComponent> ent, ref ExaminedEvent args)
                 args.PushMarkup(Loc.GetString(severity.ExamineText));
             }
 
+
             // text based on ID card
             List<ProtoId<DepartmentPrototype>>? departments = null;
+            var jobId = "";
+
             if (_id.TryFindIdCard(args.Examiner, out var id))
             {
                 departments = id.Comp.JobDepartments;
+                if (id.Comp.JobTitle is not null)
+                {
+                    jobId = id.Comp.JobTitle;
```

Comment:
use the localized job title and compare it to the prototypes localized name, isntead of comparing IDs
what you did is basically making both datafields the same

Reply:
I couldn't get the localizedJobTitle to network properly on my own, supposedly because it's technically a function per one of the people helping me in the discord. Are you asking me to get the data from the localizedJobTitle field, or just to localize the jobTitle field, and localize the JobDepartments string I'm checking it against?

Reply:
LocalizedJobTitle cannot be networked since it is a function, but both client and server already know it. The problem is likely _jobTitle not being networked.

Reply:
Should be fixed. I'm not 100% sure what you meant about making both datafields the same, the old system properly disallowed and allowed the correct contraband to the correct jobs, but I've swapped to using the localized versions so that it works properly with localization and shouldn't break. I've tested it, and it seems to give the correct responses in the correct environments now as well.